Doc/library/math.rst

```diff
@@ -247,6 +249,30 @@ Floating point arithmetic
    .. versionadded:: 3.13
 
 
+.. function:: fmax(x, y, /)
+
+   Get the larger of two floating-point values, treating NaNs as missing data.
+
+   When both operands are (signed) NaNs or zeroes, return ``nan`` and ``0``
+   respectively and the sign of the result is implementation-defined, that
+   is, :func:`!fmax` is not required to be sensitive to the sign of such
```

Comment:
```suggestion
   is, :func:`fmax` is not required to be sensitive to the sign of such
```

Reply:
I don't think we need to have the link again as we're already in the function.